modules/cco-ccoctl-creating-at-once.adoc

```diff
@@ -34,60 +38,148 @@ ifdef::google-cloud-platform[]
+----
++
+[NOTE]
+====
+Ensure that the architecture of the `$RELEASE_IMAGE` matches the architecture of the environment in which you will use the `ccoctl` tool.
+====
+
+. Extract the `ccoctl` binary from the CCO container image within the {product-title} release image by running the following command:
++
+[source,terminal]
+----
+$ oc image extract $CCO_IMAGE --file="/usr/bin/ccoctl" -a ~/.pull-secret
+----
+
+. Change the permissions to make `ccoctl` executable by running the following command:
++
+[source,terminal]
+----
+$ chmod 775 ccoctl
+----
+
+.Verification
+
+. To verify that `ccoctl` is ready to use, display the help file by running the following command:
++
+[source,terminal]
+----
+$ ccoctl --help
+----
+
+.Output
+
++
+[source,terminal]
+----
+OpenShift credentials provisioning tool
+
+Usage:
+  ccoctl [command]
+
+Available Commands:
+  alibabacloud Manage credentials objects for alibaba cloud
+  aws          Manage credentials objects for AWS cloud
+  gcp          Manage credentials objects for Google cloud
+  help         Help about any command
+  ibmcloud     Manage credentials objects for IBM Cloud
+  nutanix      Manage credentials objects for Nutanix
+
+Flags:
+  -h, --help   help for ccoctl
+
+Use "ccoctl [command] --help" for more information about a command.
+----
+
+== Using the `ccoctl` program
+
+
 .Prerequisites
 
 You must have:
 
 * Extracted and prepared the `ccoctl` binary.
-ifdef::alibabacloud-default,alibabacloud-customizations[]
+ifdef::alibabacloud-default,alibabacloud-customizations,alibabacloud-vpc[]
 * Created a RAM user with sufficient permission to create the {product-title} cluster.
 * Added the AccessKeyID (`access_key_id`) and AccessKeySecret (`access_key_secret`) of that RAM user into the link:https://www.alibabacloud.com/help/en/doc-detail/311667.htm#h2-sls-mfm-3p3[`~/.alibabacloud/credentials` file] on your local computer.
-endif::alibabacloud-default,alibabacloud-customizations[]
+endif::alibabacloud-default,alibabacloud-customizations,alibabacloud-vpc[]
 
 .Procedure
 
 . Extract the list of `CredentialsRequest` objects from the {product-title} release image:
 +
 [source,terminal]
 ifdef::aws-sts[]
+
 ----
 $ oc adm release extract \
 --credentials-requests \
 --cloud=aws \
 --to=<path_to_directory_with_list_of_credentials_requests>/credrequests \ <1>
 --from=quay.io/<path_to>/ocp-release:<version>
 ----
+
 endif::aws-sts[]
 ifdef::google-cloud-platform[]
+
+----
+$ RELEASE_IMAGE=$(./openshift-install version | awk '/release image/ {print $3}')
+----
+
 ----
 $ oc adm release extract \
 --credentials-requests \
 --cloud=gcp \
 --to=<path_to_directory_with_list_of_credentials_requests>/credrequests \ <1>
-quay.io/<path_to>/ocp-release:<version>
+$RELEASE_IMAGE
 ----
+
 endif::google-cloud-platform[]
-ifdef::alibabacloud-default,alibabacloud-customizations[]
+ifdef::alibabacloud-default,alibabacloud-customizations,alibabacloud-vpc[]
+
```

Comment:
```suggestion
```
I think this extra line is messing up the formatting. See the preview: https://55223--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_alibaba/installing-alibaba-vpc.html#using-the-ccoctl-program The code block in step 1 isn't indented or syntax-highlighted. 

I think this may also fix the broken formatting with the successive steps, but please double check in your local preview if this extra line is the issue.

Reply:
Agreed, agreed. Thank you for this, it got by me and is :-1: